deployinggpucontainers.mdx

```diff
@@ -0,0 +1,95 @@
+
+### Setup
+
+The requirements are simple, so in your preferred python environment:
+
+```bash
+git clone https://github.com/shadeform/examples.git
+cd examples/
+```
+Then in `deploy_container.ipynb` input your Shadeform API Key from [here](https://platform.shadeform.ai/settings/api), and a  `huggingface_token` if the model you need is gated (like Llama2 or Gemma).
+
+The notebook kicks off a VM request and runs the container specifically on a 1xA6000 Machine which are quite afforable on our platform. The container will download the model and run the benchmark for us. 
+
+The notebook leverages a lot of the principles found in [this post](https://docs.shadeform.ai/findingmostaffordablegpus) for using the platform to _find_ affordable GPU's.
+
+### Deploying a container
+
+Once we have an available instance, we will deploy a container that runs [lm-eval-harness](https://github.com/EleutherAI/lm-evaluation-harness/tree/main) on the model.
+Here is where we pass in environment variables, and select the docker image via our API.
+
+```python
+payload = {
+  "cloud": best_instance["cloud"],
+  "region": region,
+  "shade_instance_type": shade_instance_type,
+  "shade_cloud": True,
+  "name": "cool_gpu_server",
+  "launch_configuration": {
+    "type": "docker",
+    "docker_configuration": {
+      "image": "shadeform/lm-eval-harness",
+      "envs": [
+      	{
+	      	"name": "model",
+	      	"value": "google/gemma-7b-it"
+      	},
+      	{
+	      	"name": "tasks",
+	      	"value": "hellaswag"
+      	},
+      ]
+    }
+  }
+}
+```
+
+
+### Checking Results via the Shadeform Platform
+The first way to check the results is by watching the logs under "Running Instances", clicking our instance, and then the Logs(beta) tab. 
+Here we see a stream of the logs from the docker container. When it is done, it will look something like this
+![FinishedBenchmark](images/finishedbenchmark.png)
+
+### Checking Results inside your IDE
+At the bottom of the notebook, there is a cell that prints out the IP address of your instance.
+
+```python
+instance_response = requests.request("GET", base_url, headers=headers)
+
+print(instance_response.text)
+instance = json.loads(instance_response.text)["instances"][0]
+instance_status = instance['status']
+if instance_status == 'active':
+    print(f"Instance is active with IP: {instance['ip']}")
+else:
+    print(f"Instance isn't yet active: {instance}" )
+```
+
+We will `ssh` into the instance by leveraging this ip address and our shadeform `.pem` file.
+```bash
+ssh -i <location-of-your-pem-file> shadeform@<ip-address-copied-from-cell-output>
+```
+
+Once in, we can check the logs via the container id. The `-a` flag is if it has already finished.
+```bash
+docker ps -a
+docker container logs <container-id>
+```
+
```

Comment:
Can we add a paragraph here interpreting the results of the eval? For example, "From the results of the evaluation, we can see that the Mistral model ...."

Reply:
yes, will do, left that commentary out for simplicity